README.md

```diff
@@ -356,79 +356,17 @@ You are now ready to build!
 (vpn) $ cmake --build build
 ```
 
-
 ### How to build from source code for iOS
 
-There are two ways to build the project on iOS, using the legacy Qt build system `qmake`
-and we have also added experimental support for `cmake`.
+To build the project on iOS, we use `cmake` to configure the Xcode project files, which we
+can then open and build via the Xcode IDE.
 
 > **Note**: Due to lack of low level networking support, it is not possible to turn on
 > the VPN from the iOS simulator in Xcode.
 
-#### Building with QMake
-
-1. On iOS, we compile the app using [Xcode](https://developer.apple.com/xcode/).
-Using [xcodes](https://github.com/RobotsAndPencils/xcodes) you can install the right version: 
-``` bash
-brew install robotsandpencils/made/xcodes
-cd mozilla-vpn-client
-xcodes install 
-```
-Or manually install the required version in [.xcode-version](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/.xcode-version)
-
-2. We use `qmake` to generate the Xcode project and then we "patch" it to add
-extra components such as the wireguard, the browser bridge and so on.
-
-We patch the Xcode project using [xcodeproj](https://github.com/CocoaPods/Xcodeproj). To
-install it:
-```
-gem install xcodeproj # probably you want to run this command with `sudo`
-```
-
-3. You also need to install go >= v1.18. If you don't have it done already,
-download go from the [official website](https://golang.org/dl/).
-
-4. Copy `xcode.xconfig.template` to `xcode.xconfig`
-```bash
-cp xcode.xconfig.template xcode.xconfig
-```
-
-5. Modify the xcode.xconfig to something like:
-```
-APP_ID_MACOS = org.mozilla.macos.FirefoxVPN
-LOGIN_ID_MACOS = org.mozilla.macos.FirefoxVPN.login-item
-
-GROUP_ID_IOS = group.org.mozilla.ios.Guardian
-APP_ID_IOS = org.mozilla.ios.FirefoxVPN
-NETEXT_ID_IOS = org.mozilla.ios.FirefoxVPN.network-extension
-```
-
-6. Make sure qmake is available by setting the environment variable QT_IOS_BIN:
-
-```
-export QT_IOS_BIN=/Users/[username]/Qt/6.2.4/ios/bin
-```
-
-Then generate the Xcode project using our script (and an optional adjust token):
-
-```bash
-./scripts/macos/apple_compile.sh ios [--adjust <adjust_token>] -q ~/Qt/6.2.4/macos/bin
-```
-
-7. Xcode should automatically open. You can then run/test/archive/ship the app.
-If you prefer to compile the app in command-line mode, use the following
-command:
-```bash
-xcodebuild build CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -project "Mozilla VPN.xcodeproj"
-```
-
-#### Building with CMake (Experimental)
-
-We also support building from sources for iOS using CMake.
-
 1. On iOS, we compile the app using
 [Xcode](https://developer.apple.com/xcode/) version 12 or higher and [Qt](https://www.qt.io/download)
-version 6.3.2.
+version 6.2.4.
 
 2. Ensure rust targets for iOS development are installed.
 ```bash
```

Comment:
Our default environments (both conda and not conda) doesn't explicitly install rustup for local environments. Since this PR is updating readme instructions for a local environment, can we add this in as well?

Reply:
Yes! Excellent catch. Thank you.